coalib/bears/Bear.py

```diff
@@ -136,6 +146,23 @@ def can_detect(cls):
         """
         return cls.CAN_DETECT | cls.CAN_FIX
 
+    @classproperty
+    def maintainers(cls):
+        """
+        :return: A set containing MAINTAINERS if specified, else takes
+                 AUTHORS by default.
+        """
+        return cls.AUTHORS if cls.MAINTAINERS == set() else cls.MAINTAINERS
```

Comment:
i dont think they should be the same. Imagine the case where I create a bear today. me and my 3 friends are maintaining it. in a month i quit the project. 1 year later, even though im the author, im not still a maintainer, right?